### PR TITLE
I added freechat to the snippets

### PR DESCRIPTION
**Pull request by DamienDoumer**
## I added the freechat UI sample to snippets

I added a new sample to snippets. github.com/DamienDoumer/freechat
A sample demonstrating how to build a chat app's UI.

### CODE REVIEW

File: src/Snppts/Snippets/FreeChatSnippet.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+        public bool ContainsiOSSample => true;
+
+        public bool ContainsUWPSample => false;
+
+        public IAmAnAuthor AuthorInfo => new DamienDoumer();
+
+        public IEnumerable<Uri> ImageUris => new List<Uri>
+        {
+            new Uri("https://github.com/DamienDoumer/freechat/blob/master/images/freechatAndroidDemo.gif"),
```
Comment (jfversluis) [inline src/Snppts/Snippets/FreeChatSnippet.cs]: Can you make the images point to the actual image? Not the github page

File: src/Snppts/Snppts.csproj [hunk 2]
```diff
@@ -24,7 +24,5 @@
     <Folder Include="Views\Shared\Components\PopularCategories\" />
     <Folder Include="Views\Shared\Components\AllCategories\" />
     <Folder Include="Views\Shared\Components\RandomSnippets\" />
-    <Folder Include="Snippets\" />
```
Comment (jfversluis) [inline src/Snppts/Snppts.csproj]: Can you undo the changes to the csproj please?

**Review by jfversluis — CHANGES_REQUESTED** [review]

File: src/Snppts/Snippets/FreeChatSnippet.cs (hunk 1 above)
Comment (DamienDoumer) [reply src/Snppts/Snippets/FreeChatSnippet.cs]: Ok, done.

**Review by jfversluis — APPROVED** [review]
